safehttp/mux_test.go

```diff
@@ -199,6 +199,17 @@ func (p *claimHeaderInterceptor) SetHeader(value string) {
 	p.setValue([]string{value})
 }
 
+type panickingInterceptor struct{}
+
+func (panickingInterceptor) Before(w safehttp.ResponseWriter, _ *safehttp.IncomingRequest) safehttp.Result {
+	// Don't remove this if-statement, then "go vet" will complain about
+	// unreachable code.
+	if true {
+		panic("bad")
+	}
+	return safehttp.Result{}
```

Comment:
can you please change this code to just be a panic (without return)?

It seems a bit cleaner to me.